test/test-framework/pkg/apis/cache/v1alpha1/memcached_types.go

```diff
@@ -27,6 +27,7 @@ type MemcachedSpec struct {
 }
 type MemcachedStatus struct {
 	// Nodes are the names of the memcached pods
+	// +listType=set  //More Info: https://godoc.org/k8s.io/kube-openapi/pkg/idl
```

Comment:
> //More Info: https://godoc.org/k8s.io/kube-openapi/pkg/idl

I like the idea of including this link to the kube-openapi go doc, but anything after `// +listType=set` messes up the comment parsing for the annotation.

I looked at the [implementation of the rule](
https://github.com/kubernetes/kube-openapi/blob/0270cf2f1c1d995d34b36019a6f65d58e6e33ad4/pkg/generators/rules/idl_tag.go#L27) and the [implementation of the generator](https://github.com/kubernetes/kube-openapi/blob/b742be413d0a6f781c123bed504c8fb39264c57d/pkg/generators/extension.go#L153-L167), both of which use the `ExtractCommentTags()` function. That function [uses everything to the left of the `=` sign as the value](https://github.com/kubernetes/gengo/blob/master/types/comments.go#L55).

So by adding this comment at the end of the line, the parsed list type ends up being `set  //More Info: https://godoc.org/k8s.io/kube-openapi/pkg/idl`, not `set`, which causes this to show up in `zz_generated.openapi.go`:
```go
VendorExtensible: spec.VendorExtensible{
	Extensions: spec.Extensions{
		"x-kubernetes-list-type": "set  //More Info: https://godoc.org/k8s.io/kube-openapi/pkg/idl",
	},
},
```

Can we just include this link once in the top of the file (maybe around line 23) in another NOTE? Something like:
```
NOTE: Comment directives are necessary in certain situations to avoid API rule violations when generating OpenAPI files. See https://godoc.org/k8s.io/kube-openapi/pkg/idl to learn more.
```

Ditto for anywhere else we added this comment.

Reply:
We have a PR to add the info in the user guide and migration doc. So, I do not think that this link would be required here at all too. Could we just remove? WDYT? Also, could we not just move with the PR to update the test-framework instead of this one https://github.com/operator-framework/operator-sdk/pull/2129/files.?